utils/server/index.ts

```diff
@@ -104,13 +104,24 @@ export const OpenAIStream = async (
           const data = event.data;
 
           try {
+            if (data === '[DONE]') {
+              controller.close();
+              return;
+            }
+
             const json = JSON.parse(data);
             if (json.choices[0].finish_reason != null) {
               if (customMessageToStreamBack) {
                 const queue = encoder.encode(customMessageToStreamBack);
                 controller.enqueue(queue);
               }
 
+              if (json.choices[0].finish_reason === 'length') {
```

Comment:
How do you trigger this part?

Reply:
I try prompt 'Write me 5000 words for Climate change' doesn't work. Which should work right?